.storybook/blocks/ComponentDetails.jsx

```diff
@@ -264,13 +306,104 @@ function fetchNpmData(packageName, setnpmData, setIsLoading) {
 	}, [cache, setCache, packageName, setnpmData, setIsLoading]);
 }
 
+const fetchLogo = (brand) => {
+	switch(brand) {
+		case "npm":
+			return NpmSVG;
+		case "GitHub":
+			return GitHubSVG;
+		case "Adobe":
+			return AdobeSVG;
+	}
+
+	return;
+}
+
+export const ResourceLinkContent = ({ heading, alt, logo, href }) => {
+	if (!href) return;
+
+	return (
+		<ResourceLink href={href} className="sb-unstyled" title={alt}>
+			<ResourceIconWrapper dangerouslySetInnerHTML={{ __html: fetchLogo(logo) }} />
+			<ResourceTextWrapper>
+				{heading ? <Heading size="xs">{heading}</Heading> : ""}
+				{alt ? <Body size="s">{alt}</Body> : ""}
+			</ResourceTextWrapper>
+		</ResourceLink>
+	);
+};
+
+export const ResourceListDetails = ({ packageName, spectrumData = [], hasDocsPage, title }) => {
+	if (!packageName) return;
+
+	// reformat the component's title so split off "Components/", split off any spaces between words, and capitalize each word. It should resemble the formatting of a rootClass.
+	const reformatTitle = (component) => {
+		let reformattedComponentName = "";
+
+		const tempName = component.split("Components/").pop().split("-").pop();
```

Comment:
Splitting off the hyphen should accommodate components like in-line alert, that have a combination of hyphens and capitalization in the metadata.title and rootClass. If in-field button ever got guidelines, this approach _should_ work for that as well.

Reply:
New approach taken! reformatTitle is no longer necessary: https://github.com/adobe/spectrum-css/pull/3129/commits/c00c6b6f9959048919bcf364525e0f2bcdec7a3e